### PR TITLE
listed for sell in Orders

### PR DESCRIPTION
**Pull request by vanminh1701**

### CODE REVIEW

File: lib/src/components/Order/index.tsx [hunk 1]
```diff
@@ -34,10 +38,10 @@ export const Order: React.FC<OrderProps> = ({
     } catch (err) {
       return null;
     }
-  }, [order]);
+  }, [candyShop.baseUnitsPerCurrency, candyShop.priceDecimals, order?.price]);
 
   const onClose = useCallback(() => {
-    setSelection(null);
+    setSelection(undefined);
   }, []);
```
Comment (haVincy) [inline lib/src/components/Order/index.tsx:45]: Do we really need `useCallback` in this case?

File: lib/src/core/Orders/index.tsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ export const Orders: React.FC<OrdersProps> = ({
     number[] | undefined
   >(undefined);
 
-  const loadNextPage = (startIndex: number, limit: number) => {
+  const loadNextPage = (startIndex: number, limit: number) => () => {
```
Comment (haVincy) [inline lib/src/core/Orders/index.tsx:76]: Why we need second arrow function here..?

File: lib/src/core/Orders/index.tsx [hunk 3]
```diff
@@ -179,20 +178,18 @@ export const Orders: React.FC<OrdersProps> = ({
...
                   orders={orders}
                   walletConnectComponent={walletConnectComponent}
                   wallet={wallet}
                   candyShop={candyShop}
                   url={url}
                   hasNextPage={hasNextPage}
-                  loadNextPage={() =>
-                    loadNextPage(startIndex, ORDER_FETCH_LIMIT)
-                  }
+                  loadNextPage={loadNextPage(startIndex, ORDER_FETCH_LIMIT)}
```
Comment (haVincy) [inline lib/src/core/Orders/index.tsx:189]: It seems should be following...?
```ts
loadNextPage={(startIndex, ORDER_FETCH_LIMIT) => loadNextPage(startIndex, ORDER_FETCH_LIMIT)}
```

File: lib/src/components/Order/index.tsx (hunk 1 above)
Comment (vanminh1701) [reply lib/src/components/Order/index.tsx:45]: no, don't need it

File: lib/src/core/Orders/index.tsx (hunk 3 above)
Comment (vanminh1701) [reply lib/src/core/Orders/index.tsx:189]: startIndex, ORDER_FETCH_LIMIT come from current component, thay aren't arguments from `next` function in child componenet (InfiniteScroll)

**Pull request merged**